tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "compilerOptions": {
     "module": "commonjs",
     "noImplicitAny": true,
-    "outFile": "./build/tsc.js",
```

Comment:
What is this for?

Reply:
I couldn't run tsc to test the syntax of the definitions file without removing this